wagtail/snippets/templates/wagtailsnippets/snippets/create.html

```diff
@@ -1,7 +1,7 @@
 {% extends "wagtailadmin/base.html" %}
 {% load i18n wagtailadmin_tags %}
 {% block titletag %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}New  {{ snippet_type_name }}{% endblocktrans %}{% endblock %}
-{% block bodyclass %}page-editor create model-{{ model_opts.model_name }}{% endblock %}
+{% block bodyclass %}page-slim-header{% endblock %}
```

Comment:
not sure on the best way to manage this but I think this will be ok for now.

Maybe `:has` will help us in a few years :) https://developer.mozilla.org/en-US/docs/Web/CSS/:has

Alternatively we could look at the slim header adding a global css custom property via JS when instantiated? this could update the root/body's scroll-top-margin value.

Or... maybe it's not so bad that the scroll margin is a bit larger by default?

This body class approach just makes this more complex I think.

Reply:
True, it would probably be acceptable to have a scroll offset everywhere. I’ll make that change. `:has` would indeed be the perfect solution for this!